simpleflow/swf/process/decider/base.py

```diff
@@ -72,13 +72,11 @@ def __init__(self, executors, domain, task_list, nb_retries=3,
 
         self.nb_retries = nb_retries
 
-        Poller.__init__(
-            self,
-            domain,
-            task_list,
-            *args,    # directly forward them.
-            **kwargs  # directly forward them.
-        )
+        # Call super()'s methods: as this class uses multiple inheritance, the
+        # call is written directly so it's easier to understand.
+        # TODO: see if *args/**kwargs forwarding is really useful
+        Poller.__init__(self, domain, task_list, *args, **kwargs)
+        swf.actors.Decider.__init__(self, domain, task_list)
```

Comment:
I changed things a bit and documented the calls to the secondary parent, better?